seed/views/organizations.py

```diff
@@ -195,6 +195,79 @@ def _save_fields(org, new_fields, old_fields, is_public=False):
 _log = logging.getLogger(__name__)
 
 
+class RulesSubSerializer(serializers.Serializer):
```

Comment:
So here's a bunch of serializers.  These are never actually used to serialize data for the views.  They are simply there to render complex types in Swagger.  Maybe...we could...use these sometime to actually get data in/out of the views?  But it seems like overkill.